src/calculateRentalCost.js

```diff
@@ -29,6 +29,11 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const rent = 40;
+
+  return days < 3 ? days * rent
+    : days < 7 ? days * rent - 20
```

Comment:
```suggestion
    : days < 7 ? (days * rent) - 20
```
use parentheses or move it to a variable